features/step_definitions/call_volumes_report_steps.rb

```diff
@@ -0,0 +1,44 @@
+Given(/^I am logged in as a Pension Wise data analyst$/) do
+  User.create!(
+    name: 'Data analyst',
+    email: 'analyst@pensionwise.gov.uk',
+    uid: SecureRandom.uuid,
+    permissions: ['signin'],
```

Comment:
At this stage, everyone with access to the application will have access to the report. We might want to add more permission levels/filtering as time goes on, but the report itself doesn't contain any sensitive data so felt this was alright. And ultimately if you have access to the app but not the report you can only login and logout.. :(